tests/ap/test_ap_service.c

```diff
@@ -8,11 +8,11 @@
 #include <stdlib.h>
 #include <stdarg.h>
 #include <stddef.h>
-#include <stdint.h>
```

Comment:
I've moved `#include <stdint.h>` to be right next to `#include <cmocka.h>` to make Regex searching easier. This should also be more obvious once #116 gets merged.